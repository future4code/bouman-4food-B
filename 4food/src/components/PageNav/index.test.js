import React from 'react';
import renderer from 'react-test-renderer';
import { PageNav } from './index'

it('Snapshot PageNav', ()=> {
    const tree = renderer
    .create(<PageNav></PageNav>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})