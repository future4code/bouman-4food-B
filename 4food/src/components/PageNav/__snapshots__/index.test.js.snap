// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot PageNav 1`] = `
<div
  className="sc-bwzfXH cIAcVL"
>
  <p
    className="sc-bdVaJa kAQpGb"
  >
    Ifuture
  </p>
</div>
`;
